File: 1. Two Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        
        sums = []
        for x in range(len(nums)-1):
            for y in range(x+1,len(nums)):
                if nums[x]+nums[y]==target:
                    sums.append(x)
                    sums.append(y)
                    return sums
                
        

File: cpp_solutions/1. Two Sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        
        vector<int> ans;
        unordered_map<int,int> maps;
       
        for (int i = 0; i < nums.size();i++){
            if (maps.find(target - nums[i])!= maps.end())
            {
                ans.push_back(i);
                ans.push_back(maps[target - nums[i]]);
                    
                return ans;
            }
            
            else{
                maps[nums[i]] = i;
            }
        }
        return ans;
    }
};
